jobclient/java/src/main/java/com/twosigma/cook/jobclient/Disk.java

```diff
@@ -0,0 +1,102 @@
+package com.twosigma.cook.jobclient;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Disk {
+    /**
+     *   request - disk space guaranteed for the job
+     *   limit - max disk space the job can use
+     *   type - type of disk for the job
+     */
+    private Double _request;
+    private Double _limit;
+    private String _type;
+
+    public enum DiskType {
+        STANDARD ("standard"), PD_SSD ("pd-ssd");
+
+        private final String typeString;
+
+        /**
+         * @param typeString
+         */
+        DiskType(final String typeString) {
+            this.typeString = typeString;
+        }
+
+        /* (non-Javadoc)
+         * @see java.lang.Enum#toString()
+         */
+        @Override
+        public String toString() {
+            return typeString;
+        }
+    }
+
+    public Disk() {
+        this(null, null, null);
+    }
+
+    public Disk(Double request, Double limit, String type) {
+        this._request = request;
+        this._limit = limit;
+        this._type = type;
+    }
+
+    public void setRequest(Double request) { this._request = request; }
```

Comment:
Maybe to be consistent with the other setters:

```suggestion
    public void setRequest(Double request) { 
        this._request = request; 
    }
```

Reply:
changed